README.md

```diff
@@ -1038,22 +1049,26 @@ same headers, as received by the Device Provisioning API (this two headers are,
 outgoing requests).
 
 ### Device model
-| Attribute           | Definition                                     | Example of value                      |
-| ------------------- |:---------------------------------------------- |:------------------------------------- |
-| device_id    	      | Device ID that will be used to identify the device. | UO834IO   |
-| service             | Name of the service the device belongs to (will be used in the fiware-service header).  | smartGondor |
-| service_path        | Name of the subservice the device belongs to (used in the fiware-servicepath header). | /gardens |
-| entity_name         | Name of the entity representing the device in the Context Broker	| ParkLamplight12 |
-| entity_type         | Type of the entity in the Context Broker | Lamplights |
-| timezone            | Time zone of the sensor if it has any | America/Santiago |
-| endpoint            | Endpoint where the device is going to receive commands, if any. | http://theDeviceUrl:1234/commands  |
-| protocol            | Name of the device protocol, for its use with an IoT Manager. | America/Santiago |
-| transport           | Name of the device transport protocol, for the IoT Agents with multiple transport protocols. | MQTT |
-| attributes          | List of active attributes of the device	| `[ { "name": "attr_name", "type": "string" } ]` |
-| lazy                | List of lazy attributes of the device	| `[ { "name": "attr_name", "type": "string" } ]` |
-| commands            | List of commands of the device	| `[ { "name": "attr_name", "type": "string" } ]` |
-| internal_attributes | List of internal attributes with free format for specific IoT Agent configuration | LWM2M mappings from object URIs to attributes |
-| static_attributes   | List of static attributes to append to the entity. All the updateContext requests to the CB will have this set of attributes appended.	| `[ { "name": "attr_name", "type": "string" } ]` |
+
+The next table shows the information held in the Device resource. The table also contains the correspondence between
+the API resource fields and the same fields in the database model.
+
+| Attribute | DB Field | Definition                                     | Example of value                      |
```

Comment:
I'd suggest to use "Payload field" (or some alternative not using "attribute") to avoid confussion.